doc/field_guide__rr_lyrae.md

```diff
@@ -0,0 +1,47 @@
+## RR Lyrae (rrlyr)
+RR Lyrae are pulsating horizontal branch dA/dF stars with a mass 0.5 of Solar masses.
+They are part of the main 'instability strip' in the HR-diagram,
+where stars pulsate due to the double ionisation of He.
+The two main subtypes are `ab` and `c`,
+
+### Classification and numbers
+- Supertypes
+  - variable
+  - periodic
+  - pulsator
+- Subtypes
+  - RR Lyrae ab
+  - RR Lyrae c
+- Occurrence rate: very common, about 10<sup>5</sup> expected in ZTF data

```

Comment:
Again, would it be beneficial to mention 2 number - (1) per sq. deg in the Galaxy, (2) per sq. degree some distance away from the plane

Reply:
Same comment as above.